packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -5,6 +5,11 @@ import styles from './SlidePanel.module.scss';
 
 const cx = classNames.bind(styles);
 
+const position = {
+  start: 'start',
```

Comment:
nit: I'd make the keys UPPERCASE. 

```
const position = {
   START: 'start',
   END: 'end',
}
```

Reply:
+1 to make change to UpperCase.

we may also need to update examples and tests in doc with this new position constants to keep consistency between files.
here are the examples and tests which needs to update the position constants : 

- [DefaultSlidePanel.jsx](https://github.com/cerner/terra-framework/blob/c0ee6bbef747b0bba93a8359c58e250b333ba8d5/packages/terra-slide-panel/src/terra-dev-site/doc/example/DefaultSlidePanel.jsx#L136)

- [SlidePanelEnd.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelEnd.test.jsx#L11)
- [SlidePanelFill.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelFill.test.jsx#L12)
- [SlidePanelFullscreen.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelFullscreen.test.jsx#L11)
- [SlidePanelLarge.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelLarge.test.jsx#L11)
- [SlidePanelNoFill.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelNoFill.test.jsx#L12)
- [SlidePanelOverlay.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelOverlay.test.jsx#L11)
- [SlidePanelSmall.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelSmall.test.jsx#L11)
- [SlidePanelSquish.test.jsx](https://github.com/cerner/terra-framework/blob/4dcff701a4db4212eadd5f7e65b808b3872c11f2/packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelSquish.test.jsx#L11)

Reply:
The keys of the enum object have been changed to UpperCase.
Also, the enum values have been incorporated in the examples as well.